store/mongo/datastore_mongo.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/mendersoftware/deviceauth/store"
 
 	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
+	ctxStore "github.com/mendersoftware/go-lib-micro/store"
```

Comment:
According to https://golang.org/doc/effective_go.html#names
both names are not very good, because 
"there should be no need for underscores or mixedCaps".
ctxstore is not single-word name so there is a need for mixedCaps IMO.